dali/pipeline/operator/op_schema.h

```diff
@@ -64,6 +68,10 @@ enum class InputDevice : uint8_t {
   GPU = 2,
 };
 
+inline bool HideArgument(const std::string &name) {
+  return name.size() && name[0] == '_';
+}
```

Comment:
Can you move it inside the Schema class as a static function?
Also, the imperative name doesn't look like a boolean query.
Alternatives: ShouldHideArgument, ArgumentHidden, IsArgumentHidden or perhaps just `IsHidden(const string &arg_name)`

Reply:
good point